GA-worksheet.md

```diff
@@ -72,25 +72,25 @@
     }
 }
     ```
-
+As long as the window is 500px or smaller, the background is red.
```

Comment:
Right. The @media tag is a condition, and if the condition within the parenthesis is true, the CSS inside will be activated.